collection-scripts/gather

```diff
@@ -67,5 +67,8 @@ oc adm inspect --dest-dir must-gather "${group_resources_text}"
 # Gather HAProxy config files
 /usr/bin/gather_haproxy_config
 
+# Gather Monitoring metrics
+/usr/bin/gather_monitoring
```

Comment:
gather_monitoring is already executed above, at line 49

Reply:
@dgrisonnet good catch. Thanks. Removed!